mm.c

```diff
@@ -59,26 +59,32 @@ team_t team = {
 /* rounds up to the nearest multiple of ALIGNMENT */
 #define ALIGN(size) (((size) + (ALIGNMENT-1)) & ~(ALIGNMENT-1))
 
-// Pack a size and allocated bit into a wBIT_ord
+// Pack a size and allocated bit into a BIT_word
 #define PACK(size, status) ((size<<STATUS_BIT_SIZE) | (status))
 
 /* Round up to even */
 #define EVENIZE(x) ((x + 1) & ~1)
 
-// Read the size and allocation bit from address p 
+// Read the size and allocation bit from address p
 #define GET_SIZE(p)  ((GET_WORD(p) & ~GET_MASK(STATUS_BIT_SIZE)) >> STATUS_BIT_SIZE)
 #define GET_STATUS(p) (GET_WORD(p) & 0x1)
 
-// Address of block's footer 
+// Address of block's footer
 // Take in a pointer that points to the header
 #define FTRP(header_p) ((char **)(header_p) + GET_SIZE(header_p) + 1)
 
 // Get total size of a block
-// Size indicates the size of the free space in a block 
+// Size indicates the size of the free space in a block
 // Total size = size + size_of_header + size_of_footer = size + D_WORD_SIZE
 // p must point to a header
 #define GET_TOTAL_SIZE(p) (GET_SIZE(p) + HDR_FTR_SIZE)
 
+// Given pointer to current block, return pointer to header of previous block
+#define PREV_BLOCK_IN_MEM(header_p) ((char **)(header_p) - GET_SIZE((char **)(header_p) - 1) - HDR_FTR_SIZE)
```

Comment:
I tried to achieve that with the `MEM` ending. If I change it to `HEAP` will it be clearer?